backend/pkg/api/connect/service/common/mapper.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// Package common contains common types and mapping utility functions that are used across
+// proto packages in endpoint handlers.
+package common
+
+import (
+	"fmt"
+
+	"github.com/twmb/franz-go/pkg/kmsg"
+
+	v1alpha1 "github.com/redpanda-data/console/backend/pkg/protogen/redpanda/api/dataplane/v1alpha1"
+)
+
+// KafkaClientMapper is a utility to map common types between the Kafka Client and the
+// proto types that are used for requests and responses.
+type KafkaClientMapper struct{}
+
+// ConfigTypeToProto maps the kmsg.ConfigType enum to the proto enum.
+func (*KafkaClientMapper) ConfigTypeToProto(configType kmsg.ConfigType) (v1alpha1.ConfigType, error) {
+	switch configType {
+	case kmsg.ConfigTypeBoolean:
+		return v1alpha1.ConfigType_CONFIG_TYPE_DOUBLE, nil
```

Comment:
just checking... Is this correct, the case is boolean, but we return double?

Reply:
No not correct, good find. Thanks!